Fragments/PartonShower/powheg-herwig7.dat

```diff
@@ -0,0 +1,39 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Generator.Herwig7Settings.Herwig7LHECommonSettings_cfi import *
+from Configuration.Generator.Herwig7Settings.Herwig7StableParticlesForDetector_cfi import *
+from Configuration.Generator.Herwig7Settings.Herwig7CH3TuneSettings_cfi import *
+from Configuration.Generator.Herwig7Settings.Herwig7LHEPowhegSettings_cfi import *
+from Configuration.Generator.Herwig7Settings.Herwig7PSWeightsSettings_cfi import *
+from Configuration.Generator.Herwig7Settings.Herwig7_7p1SettingsFor7p2_cfi import *
```

Comment:
not sure what this really does, are you aware of this?

Reply:
Uhm that is loading the Herwig CH3 tuning right? Now what I'm not so sure is the `Herwig7_7p1SettingsFor7p2_cfi`, not sure if that's the latest.

Reply:
we should ask dominic

Reply:
Hi, just to add here, the `Herwig7_7p1SettingsFor7p2` set a number of UE settings which changed when going from Herwig 7.1 to 7.2 back to their 7.1 values, since this was the release used to make the CH tunes

Reply:
@Dominic-Stafford wouldn't it make sense to add this to the CH settings? It will work with the old versions where this was already default and still with the new ones.  We could make an entry in the fragment that these are not actual settings of the tune but default ones that have changed and where used to derive the tune. The current name is a bit confusing. I will still work with 7.3 and it is related to the tune not the general program, so people might only want it if they are using CH3.

Reply:
Yes, maybe this would make sense, though it might also cause confusion for people taking parameters from old cards if we remove it. Maybe first I should ask Emyr is there was a motivation for making it a separate block? For 7.3 this might need to be expanded as maybe there will be further parameters which should be set back to their 7.1 values